integrations/common/http.go

```diff
@@ -19,30 +20,84 @@ const (
 	ContentTypeForm            = "application/x-www-form-urlencoded"
 	ContentTypeJSON            = "application/json"                // Accept
 	ContentTypeJSONCharsetUTF8 = "application/json; charset=utf-8" // Content-Type
+
+	HTTPTimeout = 3 * time.Second
+	HTTPMaxSize = 1 << 23 // 2^23 bytes = 8 MiB
 )
 
+func HTTPDeleteJSON(ctx context.Context, u, auth string, payload any) ([]byte, error) {
+	if s, ok := payload.(string); ok {
+		return httpRequest(ctx, http.MethodDelete, u, auth, ContentTypeJSONCharsetUTF8, []byte(s))
+	}
+
+	body, err := json.Marshal(payload)
+	if err != nil {
+		return nil, fmt.Errorf("failed to marshal JSON payload: %w", err)
+	}
+	return httpRequest(ctx, http.MethodDelete, u, auth, ContentTypeJSONCharsetUTF8, body)
+}
+
+func HTTPGet(ctx context.Context, u, auth string) ([]byte, error) {
```

Comment:
Don't see a lot of value in this convenience function.

Reply:
I see your point, but I'll do this in a separate PR, it affects a lot of places